### DISCUSSION TITLE
How to stop pandoc to access url when converting html to markdown

### DISCUSSION THREAD

**Discussion by link89** [jgm/pandoc#10449] [Q&A]
When I am using pandoc to converting html to markdown,  I find it keeps print warning like the below
```
[WARNING] Could not fetch resource https://li.protechts.net/index.html?ts=1733822728348&r_id=AAYo5w8RaAJF1bbpPoM6vQ%3D%3D&app_id=PXdOjV695v&uc=scraping&d_id=864c994318c23e86864519a930e0f0c426d8a40b35270cd6cb28ae330389ee53: File https://li.protechts.net/index.html?ts=1733822728348&r_id=AAYo5w8RaAJF1bbpPoM6vQ%3D%3D&app_id=PXdOjV695v&uc=scraping&d_id=864c994318c23e86864519a930e0f0c426d8a40b35270cd6cb28ae330389ee53 
```

I don't need to have pandoc to process thoes links all. The below is the command I use:

```powershell
pandoc --sandbox -f html-native_divs-native_spans -t markdown "out\cvs\Shasha Chong-Caltech\cv-www.linkedin.com_in_shashachong.html" -o "out\cvs\Shasha Chong-Caltech\cv-www.linkedin.com_in_shashachong.html.md"
```

Is there any option that I can use to stop pandoc from accessing url?

**Comment by jgm** [comment]
Pandoc should not normally try to fetch these resources in a md -> html conversion, unless you specify `--embed-data-files` or `--self-contained`.  Can you give a simple case we can use to reproduce this (upload a file or just quote a small md snippet that suffices to reproduce the issue).

**Reply by link89** [reply]
You may try with any linkedin page. I think I figure out the issue. It looks like pandoc will load the iframe url.

**Reply by jgm** [reply]
Ah yes, it will do that if there's an iframe.  In your case this fails because you specify `--sandbox`, which prevents the IO.
